### PR TITLE
♻️Maintenance: webserver openapi users now auto-generated

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- [x] openapi-users.yaml is now auto-generated via fastapi
- [x] openapi-user.yaml --> openapi-users.yaml
- [x] removed unused PUT /me/profile

## Related issue/s

- pre-requisite of https://github.com/ITISFoundation/osparc-simcore/pull/4417
- relates to https://github.com/ITISFoundation/osparc-issues/issues/675

Bonus:
- fixes https://github.com/ITISFoundation/osparc-simcore/issues/2991

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
pair reviewed

File: services/docker-compose.yml [hunk 1]
```diff
@@ -270,6 +270,7 @@ services:
       DIRECTOR_PORT: ${DIRECTOR_PORT:-8080}
       DIRECTOR_V2_HOST: ${DIRECTOR_V2_HOST:-director-v2}
       DIRECTOR_V2_PORT: ${DIRECTOR_V2_PORT:-8000}
+      GUNICORN_CMD_ARGS: --timeout=90
```
Comment (odeimaiz) [inline services/docker-compose.yml:273]:
```suggestion
      GUNICORN_CMD_ARGS: --timeout=180
```

Comment (odeimaiz) [reply services/docker-compose.yml:273]: Merci :)

**Review by odeimaiz — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]
looks good :+1:

**Pull request merged**